pages/mobile/base.py

```diff
@@ -64,6 +64,12 @@ def wait_notification_box_visible(self):
     def wait_notification_box_not_visible(self):
         self.wait_for_element_not_visible(*self._notification_locator)
 
+    def login(self, user):
+        credentials = isinstance(user, MockUser) and user
```

Comment:
The same problem as above exists here. What happens if `isinstance(user, MockUser)` returns `False`? If that is a possibility then you need a code path to deal with that.